docs/architecture/adr-008-dCERT-group.md

```diff
@@ -0,0 +1,117 @@
+# ADR 008: Decentralized Computer Emergency Response Team (dCERT) Group
+
+## Changelog
+
+- 2019 Jul 31 : Initial Conception
+
+## Context
+
+In order to reduce the number of parties involved with handling sensitive
+information in an emergency scenario, we propose the creation of a
+specialization group named The Decentralized Computer Emergency Response Team
+(dCERT).  Initially this group's role is intended to serve as coordinators
+between the validators, bug-hunters, developers, and other members of the wider
+community.  During a crisis time, the dCERT group would facilitate and relay
+input from a variety of stakeholders to the developers who are actively
+devising a patch to the software, this way sensitive information does not need
+to be publicly disclosed while as some input from the community can still be
+gained. 
+
+Additionally, a special privilege is proposed for the dCERT group:
+the capacity to "circuit-break" (aka. temporarily disable)  a particular
+message path. 
+
+In the future it is foreseeable that the community may wish to expand the roles
+of dCERT with further responsibilities such as the capacity to "pre-approve" a
+security update hard-fork on behalf of the community prior to a full community
+wide vote whereby the sensitive information would be revealed prior to a
+vulnerability being patched on the live network.  
+
+## Decision
+
+The dCERT group is proposed to include an implementation of a `SpecializationGroup`
+as defined in [ADR 007](./adr-007-specialization-groups.md). This will include the 
+implementation of: 
+ - continuous voting
+ - slashing due to breach of soft contract
+ - revoking a member due to breach of soft contract
+ - compensation stipend from the community pool or other means decided by
+   governance
+
+This system necessitates the following new parameters: 
+ - blockly stipend allowance per dCERT member (suggested ~$10K/yr of atoms)
+ - maximum number of dCERT members (suggested 7) 
+ - required staked slashable tokens for each dCERT member (suggested $10K of atoms)
+ - unbonding time for dCERT staked token (suggested 3 week)
+ - quorum for suspending a particular member (suggested 5/7) 
+
+These parameters are expected to be implemented through the param keeper such 
+that governance may change them at any given point. 
+
+### Continuous Voting Electionator
+
+An `Electionator` object is to be implemented with continuous voting is to be
+implemented with the following specifications:
+ - All delegation addresses may submit votes at any point which updates their 
+   preferred representation on the dCERT group. 
+ - Preferred representation may be arbitrarily split between address (ex. 50%
```

Comment:
Should be able to determine different voting power for different types of decisions. For example, Member A is security expert and gets outsize share of voting power on circuit-breaking, security-patch upgrades but has equal voting power on non-security issues

Reply:
As mentioned in 007:

> This type of community organization paves the way for
individual stakeholders to delegate votes by issue type, if in the future
governance proposals include a field for issue type.

I think for the time being this addition would be an overly complexity. We've made simplifications like this all throughout the hub for instance currently delegators don't even get to choose different validators between who secures their stake vs who's vote they automatically vote with when they don't vote for a proposal themselves (although arguably it would be beneficial to create this separation). 

In the future, I could imagine further specification to special permissions within the CERT group as a good idea, or actually even better distinct CERT groups with distinct permissions which need to coordinate with each other, but I think for the time being, equal voting power for everything involved in the CERT group makes the most sense to minimize complexity (and maximize comprehensibility to the community as we're rolling this feature out)